wasm/module_test.go

```diff
@@ -14,21 +14,29 @@ import (
 )
 
 func TestReadModule(t *testing.T) {
-	fnames, err := filepath.Glob("testdata/*wasm")
+	fnames, err := filepath.Glob(filepath.Join("testdata", "*.wasm"))
 	if err != nil {
 		t.Fatal(err)
 	}
 	for _, fname := range fnames {
-		raw, err := ioutil.ReadFile(fname)
-		if err != nil {
-			t.Error(err)
-			continue
-		}
-		r := bytes.NewReader(raw)
-		_, err = wasm.ReadModule(r, nil)
-		if err != nil {
-			t.Errorf("error reading module %q: %v", fname, err)
-			continue
-		}
+		testModule(t, fname)
```

Comment:
I thought that with refactoring into `testModule`, the `return`s would be more natural-looking Go code.

I put a `defer` for good measure :)